flujo/cli/helpers.py

```
@@ -35,6 +35,25 @@
 
+def print_rich_or_typer(msg: str, *, style: Optional[str] = None, stderr: bool = False) -> None:
+    """Print a message using Rich when available, else fall back to typer.echo.
+
+    - Catches only ModuleNotFoundError for the Rich import to avoid masking bugs.
+    - Does not swallow other exceptions from Console.print, preserving error context.
+    """
+    try:
+        from rich.console import Console as _C
+
+        _C(stderr=stderr).print(msg, style=style)
+        return
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)